### DISCUSSION TITLE
How do I see all the Targets I've played?

### DISCUSSION THREAD

**Discussion by GregJacobs82** [cssbattle/forum#254] [General]
Newbie to CSSBattle here... so I apologize in advance if this is an obvious answer, or has already been referenced in another discussion (I searched and didn't find anything!)

How do I simply see all the Targets I've played / submitted an answer to?

**Comment by Beowolve** [comment]
On the bottom left is a button "My solutions" that will list your last 5 submissions. It was different in the past, where you could see all your submissions, but that is a pro feature now.
I keep notes for every target I play anyway, so without a pro subscriptions I would suggest just doing this.

**Comment by chinchang** [comment]
Hey @GregJacobs82 Your question triggered us to think that there is actually no way to see what targets you have played already. So today we are opening `/me` page to all players.
You can go to [https://cssbattle.dev/me](https://cssbattle.dev/me) and see your basic target stats. And if you want to see some ["Advanced analytics"](https://cssbattle.dev/blog/advanced-analytics), you can [Upgrade to PRO](https://cssbattle.dev/pro).

Cheers!
